specs/posix/ssh_keys.table

```diff
@@ -0,0 +1,15 @@
+
+table_name("ssh_keys")
+
+description("Returns the id_rsa file for users and whether or not they are encrypted")
+
+schema([
+    Column("uid", BIGINT, "The local user that owns the ssh_keys file",
+      additional=True),
+    Column("key_file", TEXT, "Path to key file"),
```

Comment:
Would you mind making `path` instead of `key_file`? With `path` we could join against other tables in osquery.